docker/Dockerfile

```diff
@@ -0,0 +1,62 @@
+ARG DISTRO
+FROM ubuntu:${DISTRO} as gridinit-base
+
+LABEL maintainer "Jean-Francois SMIGIELSKI <jf.smigielski@gmail.com>"
+
+ARG DEBIAN_FRONTEND=noninteractive
+ARG DEBCONF_NONINTERACTIVE_SEEN=true
+
+RUN set -ex && \
+    echo "tzdata tzdata/Areas select Europe" > /tmp/preseed.txt; \
+    echo "tzdata tzdata/Zones/Europe select Brussels" >> /tmp/preseed.txt; \
+    debconf-set-selections /tmp/preseed.txt && \
+    apt-get update -q && \
+    apt-get install -y --no-install-recommends tzdata pkg-config software-properties-common
+
+
+#------------------------------------------------------------------------------#
+# Build and Install the exotic dependencies of gridinit
+
+FROM gridinit-base as gridinit-build-deps
+
+RUN set -ex && \
+    # Deps
+    apt-get update -q && \
+    apt-get install -y --no-install-recommends git autoconf automake autotools-dev build-essential m4 libtool && \
+    # Checkout
+    cd /tmp && \
+    git clone https://github.com/open-io/libdill.git && \
```

Comment:
To save bandwidth, we can use `--single-branch -b release` (since e738f9661a5f63c219ea3a7e8627237b5fa0d9c0 is in the release branch).